src/test/java/teammates/test/cases/automated/FeedbackSessionClosedReminderTest.java

```diff
@@ -1,185 +0,0 @@
-package teammates.test.cases.automated;
```

Comment:
Instead of removing the test files entirely, could we just modify the Test annotations to `@Test(enabled = false)` to temporarily disable them? This will help keep the git history in the project at least, and make it harder to miss out on re-adding the test file